recipe/meta.yaml

```diff
@@ -44,10 +43,10 @@ test:
     - pdfstream.vend
     - pdfstream.visualization
   commands:
-    - pdfstream --help
+    - pdfstream -- --help
 
 about:
-  home: https://github.com/st3107/pdfstream
+  home: https://github.com/{{ org }}/pdfstream
```

Comment:
I'd recommend to put the org name explicitly here, so we can just copy the URL.

Reply:
Also, I'd suggest keeping the original, case-sensitive version: https://github.com/xpdAcq/PDFstream.

Reply:
Done.